### ISSUE TITLE
MS Excel 365: jump to a certain cell address results in loss of focus

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#11238]
### Steps to reproduce:
1. Open NVDA and an empty sheet in MS Excel (it might be reproducible on several versions)
2. Press ctrl+g
3. Type the cell address you want to jump to (i.e. g50)

### Actual behavior:
NVDA reports only the window title and you cannot navigate through the table anymore. You have to press alt+tab and turn back to the excel sheet to regain focus.

### Expected behavior:
NVDA should report the cell you jumped to (i.e. g50) and should perform navigation as usual when pressing arrow keys.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-20256,f2cbfec8

#### Windows version:
Windows 10 1909 update

#### Name and version of other software in use when reproducing the issue:
MS Excel 365, MSO (16.0.12827.20200) 32-Bit

#### Other information about your system:
I also tried to maximize the window, the issue still persists.

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2020.1 the same issue

#### If addons are disabled, is your problem still occuring?
yes, but not as often. It seems it is strongly related to bluetooth audio add-on. When this add-on is enabled, I can reproduce it much more often.
cc: @mltony 
It seems also the IBM TTS libraries and the IBM TTS driver is also contributing to this issue. cc: @davidacm 

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
yes, no difference.

#### Other information
It might be also related to the fact that NVDA does not detect if the focused cell is visible when jumping (i.e. from g1 to k800) and this might result in no focus reporting. Maybe @lukaszgo1 and @codeofdusk could also add some thoughts here.

**Comment by ruifontes** [comment]
Not reproduced here.

Windows 10 1909.

NVDA Alpha20256

Excel 2019 from Office 365


Rui Fontes


Às 20:17 de 08/06/2020, Adriani90 escreveu:

**Comment by Adriani90** [comment]
Rui did you try multiple times, jumping back and forth between cells, using big distance? I.e. from k1 to h400, from l2 to a200, from  a200 to b3?

 

Also it is not happening very often when you have all add-ons disabled, but it happens also then. As I said, it seems bluetooth audio and IBM TTS driver are causing more reliable results in my case at least.

 

 

Von: ruifontes <notifications@github.com> 
Gesendet: Montag, 8. Juni 2020 21:26
An: nvaccess/nvda <nvda@noreply.github.com>
Cc: Adriani90 <adriani.botez@googlemail.com>; Author <author@noreply.github.com>
Betreff: Re: [nvaccess/nvda] MS Excel 365: jump to a certain cell address results in loss of focus (#11238)

 

Not reproduced here.

Windows 10 1909.

NVDA Alpha20256

Excel 2019 from Office 365


Rui Fontes


Às 20:17 de 08/06/2020, Adriani90 escreveu:

**Comment by Adriani90** [comment]
This might also be strongly related to #5413 and #6453.

**Comment by ruifontes** [comment]
Now it happens frequently...
Nothing change, but now it happens almost everytime... Even with IBM TTS deactivated and using Eloquence SAPI5...
Very strange...

**Comment by Adriani90** [comment]
Still reproducible in NVDA 2020.3 and Office 365.

cc: @michaelDCurran

**Comment by Adriani90** [comment]
This seems fixed in NVDA 2022.4. I cannot reproduce it anymore. Closing as works for me. @ruifontes if this is still reproducible for you, please comment and we can reopen it.
